### PR TITLE
fix: show warning on big period, is forecast enable check segment

### PR DESCRIPTION
**Pull request by hackerstanislav**
JIRA: F1-316, F1-319

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18914/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20642/ : FAILURE in 9m 25s

File: libs/sdk-ui-charts/src/forecast.ts [hunk 1]
```diff
@@ -19,10 +19,14 @@ export function isForecastEnabled(
             const buckets = insightBuckets(insight);
             const measures = bucketsFind(buckets, (b) => b.localIdentifier === BucketNames.MEASURES);
             const trends = bucketsFind(buckets, (b) => b.localIdentifier === BucketNames.TREND);
+            const segments = bucketsFind(buckets, (b) => b.localIdentifier === BucketNames.SEGMENT);
 
             //TODO: s.hacker check if the trend is date attribute somehow
```
Comment (kandl) [inline libs/sdk-ui-charts/src/forecast.ts:24]: Some unresolved comment?

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/2149/ : SUCCESS in 2m 02s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20644/ : SUCCESS in 9m 35s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1947/ : SUCCESS in 10m 47s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1927/ : UNSTABLE in 9m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5446/ : SUCCESS in 6s

**Pull request merged**